sansa-query-spark/src/main/scala/net/sansa_stack/query/spark/dof/node/Helper.scala

```diff
@@ -0,0 +1,69 @@
+package net.sansa_stack.query.spark.dof.node
+
+import java.io.{ File, PrintWriter }
+
+import org.apache.jena.graph.{ Node, Triple }
+import org.apache.spark.rdd.RDD
+
+object Helper {
```

Comment:
If this class and its methods are necessary to function, we should consider of moving it to the `sansa-query-common` module or add these functionality on SANSA-RDF model.

Reply:
after deleting the logging, only four methods remain. they are used mostly to support the vertical partition schema and return the correct triple part